bluebottle/clients/middleware.py

```diff
@@ -19,17 +19,13 @@ def set_tenant(self, tenant):
         # Always default to standard django settings, e.g.
         # when tenant has no specific config, has no directory
         # or when no MULTI_TENANT_DIR is configured
-        try:
-            for settings_file in ['properties', 'secrets']:
-                props_mod = safe_join(settings.MULTI_TENANT_DIR,
-                                      tenant.client_name,
-                                      "{0}.py".format(settings_file))
-                # try to load tenant specific properties. We're using execfile since tenant
-                # directories are not python packages (e.g. no __init__.py)
-                execfile(props_mod, dict(settings=settings),
-                         self.tenant_properties)
-        except (ImportError, AttributeError, IOError):
-            pass
+        props_mod = safe_join(settings.MULTI_TENANT_DIR,
```

Comment:
It should just be there I think. It's better to get a blunt error than it to proceed without notice.